.github/workflows/main.yml

```diff
@@ -205,63 +205,123 @@ jobs:
     - build-wrappers-linux
     - build-wrappers-windows-uwp
     steps:
-    - name: Fetch Artifacts MacOS
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.0.2
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Set version suffix
+      run: |
+        if [[ $GITHUB_REF == "master" ]]
+        then
+          echo "BUILD_SUFFIX=alpha.$GITHUB_RUN_ID" >> $GITHUB_ENV
```

Comment:
Why store those in env variables rather than step outputs? I kind of dislike relying on env variables too much.

Reply:
It just seemed the consensus on how to do it and if I understand correctly, but I need to investigate further, the same env vars can be used in other jobs within the same runner without the need of: making an output for a step and and output for the job that reads the output of the step; or simply duplicated the step wherever needed.

Reply:
According to this post: https://github.community/t/sharing-a-variable-between-jobs/16967/2, we can't share environment variables between jobs, so those need to be returned as outputs from the job anyway. Or has that changed since 2019?